source/vistas/ui/controls/gl_camera.py

```diff
@@ -116,6 +120,10 @@ def Reposition(self):
 
 
 class GLCameraControls(wx.EvtHandler):
+    """
+    Event handler for controlling the the currently enabled CameraInteractor for a given GLCanvas. Handles window
```

Comment:
The fact that this is an event handler (in the wx sense) does not seem like the most pertinent thing to know about this class. I.e., handling events isn't its main function, its how it goes about performing its main function.